azkaban-web-server/src/main/java/cloudflow/services/ProjectService.java

```diff
@@ -0,0 +1,12 @@
+package cloudflow.services;
+
+import azkaban.user.User;
+import cloudflow.models.Project;
+import java.util.List;
+
+public interface ProjectService {
+
+  String create(Project project, User user);
+  Project get(String projectId);
```

Comment:
Consider renaming these methods to _createProject_ and _getProject_ to maintain consistency.